tests/test_pytest_pensieve.py

```diff
@@ -98,6 +99,36 @@ def test_memory_alloc_fails():
     assert result.ret == ExitCode.OK
 
 
+@pytest.mark.parametrize(
+    "memlimit, mem_to_alloc",
+    [(5, 100), (10, 200)],
+)
+def test_memray_with_junit_xml_error_msg(
+    pytester: Pytester, memlimit: int, mem_to_alloc: int
+):
+    xml_output_file = pytester.makefile(".xml", "")
+    pytester.makepyfile(
+        f"""
+        import pytest
+        from memray._test import MemoryAllocator
```

Comment:
@pablogsal can we expose this publicly instead of doing this private import?

Reply:
Nop, this is (and will remain as) a testing utility. I consider it the same as doing this import in memray itself.

Reply:
👍